### PR TITLE
ci: Add style check

### PR DESCRIPTION
**Pull request by evelikov**
Closes: https://github.com/intel/libva/issues/38
Closes: https://github.com/intel/libva/issues/393

Automate things instead of doing it manually as mentioned in https://github.com/intel/libva/pull/631

/cc @sivileri for the dos2unix changes

### CODE REVIEW

**Comment by sivileri** [comment]
The libva-win32 related style changes look good to me

**Review by dvrogozh — CHANGES_REQUESTED** [review]

File: .github/workflows/style.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  style-check:
+    runs-on: ubuntu-22.04
+    steps:
+    - uses: actions/checkout@v3
+    - name: install prerequisites
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y \
+          astyle \
+          git
```
[the remark this answers is not in the document]
Comment (evelikov) [reply .github/workflows/style.yml]: Thanks fixed.

**Review by dvrogozh — APPROVED** [review]
looks good

**Review by XinfengZhang — APPROVED** [review]
LGTM, so, first, run astyle , then check whether there are file changed by astyle, and report pass/failed.

**Pull request merged**